### PR TITLE
OSSMDOC-199 Release Notes for 1.1.12 and 2.0.2 releases.

### PR DESCRIPTION
**Pull request by JStickler**
Also addresses [OSSMDOC-211](https://issues.redhat.com/browse/OSSMDOC-211).

### CODE REVIEW

**Comment by JStickler** [comment]
@tvieira  or @yxun , could you review and approve the release notes please?

File: modules/ossm-rn-deprecated-features.adoc [hunk 1]
```diff
@@ -15,7 +15,7 @@ Deprecated functionality is still included in {product-title} and continues to b
 
 == Deprecated features {ProductName} 2.0
 
-The Mixer component is deprecated and will be removed in a future release.  While using Mixer for implementing extensions is still supported in release 2.0, you should be migrating your extensions to the new link:https://istio.io/latest/blog/2020/wasm-announce/[WebAssembly] mechanism.
+The Mixer component was deprecated in release 2.0 and was removed in release 2.1.  While using Mixer for implementing extensions was still supported in release 2.0, extensions should have been migrated to the new link:https://istio.io/latest/blog/2020/wasm-announce/[WebAssembly] mechanism.
```
Comment (yxun) [inline modules/ossm-rn-deprecated-features.adoc:18]: @JStickler line 18, we haven't released 2.1 yet. So should we replace "was removed in release 2.1" with "will be removed in release 2.1" ?

Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc]: Good catch, I was getting ahead of myself.  Fixed.

**Comment by JStickler** [comment]
@SNiemann15, I added your note in three places, ossm-supported-configurations, ossm-rn-known-issues, and jaeger-rn-known-issues.

**Review by yxun — APPROVED** [review]

File: modules/ossm-rn-fixed-issues-1x.adoc [hunk 2]
```diff
@@ -83,6 +52,10 @@ $ oc delete crd challenges.certmanager.k8s.io
 
 * link:https://issues.jboss.org/browse/MAISTRA-348[MAISTRA-348] OpenShift 4 Beta on AWS does not support ingress gateway traffic on ports other than 80 or 443.  If you configure your ingress gateway to handle TCP traffic with a port number other than 80 or 443, you have to use the service hostname provided by the AWS load balancer rather than the OpenShift router as a workaround.
 
+* link:https://issues.jboss.org/browse/MAISTRA-193[MAISTRA-193] Unexpected console info messages are visible when health checking is enabled for citadel.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1821432[Bug 1821432] Toggle controls in {product-title} Control Resource details page do not update the CR correctly. UI Toggle controls in the Service Mesh Control Plane (SMCP) Overview page in the {product-title} web console sometimes update the wrong field in the resource. To update a SMCP, edit the YAML content directly or update the resource from the command line instead of clicking the toggle controls.
```
Comment (neal-timpe) [inline modules/ossm-rn-fixed-issues-1x.adoc:57]: s/To update a SMCP/To update a `ServiceMeshControlPlane` resource
Or /an SMCP/

File: modules/jaeger-rn-fixed-issues.adoc [hunk 3]
```diff
@@ -15,4 +15,12 @@ Fix - What did we change to fix the problem?
...
+
 * link:https://issues.jboss.org/browse/TRACING-1631[TRACING-1631] Multiple Jaeger production instances, using same name but within different namespaces, causing Elasticsearch certificate issue.  When multiple service meshes were installed, all of the Jaeger Elasticsearch instances had the same Elasticsearch secret instead of individual secrets, which prevented the Elasticsearch Operator from communicating with all of the Elasticsearch clusters.
+
+* link:https://issues.redhat.com/browse/TRACING-1300[TRACING-1300] Failed connection between Agent and Collector when using Istio sidecar. An update of the Jaeger Operator enabled TLS communication by default between a Jaeger sidecar agent and the Jaeger Collector.
+
+* link:https://issues.redhat.com/browse/TRACING-1208[TRACING-1208] Authentication "500 Internal Error" when accessing Jaeger UI. When trying to authenticate to the UI using OAuth, I get a 500 error because oauth-proxy sidecar doesn't trust the custom CA bundle defined at installation time with the additionalTrustBundle.
```
Comment (neal-timpe) [inline modules/jaeger-rn-fixed-issues.adoc:24]: Should additionalTrustBundle be a codephrase?

**Review by neal-timpe — APPROVED** [review]

**Review by jeana-redhat — APPROVED** [review]

**Pull request merged**